switch/switch_main.ml

```diff
@@ -319,8 +319,9 @@ let main ({ Config.daemonize; path; pidfile } as config) =
         end else raise (Does_not_exist dir)
       | _ ->
         () );
-    if daemonize
-    then Lwt_daemon.daemonize ();
+
+    info "Logs redirected to syslog";
+    Lwt_log.default := Lwt_log.syslog ~facility:`Daemon ();
```

Comment:
Just a note here: Since we do not say here explicitly which log messages should be displayed, I think that means that logs of level "notice" and above will be displayed. This is fine though, no need to change anything.